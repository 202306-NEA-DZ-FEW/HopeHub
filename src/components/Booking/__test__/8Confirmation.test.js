import renderer from "react-test-renderer";

import Confirmation from "../8Confirmation";
const user = { uid: "111" };
it("renders correctly", () => {
    const tree = renderer.create(<Confirmation user={user} />).toJSON();
    expect(tree).toMatchSnapshot();
});
