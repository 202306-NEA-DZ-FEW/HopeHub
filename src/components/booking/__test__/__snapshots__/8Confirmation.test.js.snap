// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="bg-NeutralWhite min-w-screen  mb-12"
>
  <div
    className="w-full h-full px-8 lg:px-20 bg-NeutralWhite "
  >
    <div
      className="mb-3 pt-12 font-ogg font-bold text-NeutralBlack uppercase text-2xl lg:text-4xl leading-normal"
    >
      your request has been submitted!
    </div>
    <div
      className="font-poppins font-regular text-justify text-NeutralBlack text-base lg:text-lg leading-relaxed"
    >
       You will receive an email guiding you to book a date and time soon.
    </div>
    <div
      className="flex flex-col bg-white lg:w-3/5 lg:h-1/2 sm:w-full sm:h-[80%] sm:leading-tight mx-auto my-14 shadow-[4.0px_8.0px_8.0px_rgba(0,0,0,0.42)] rounded-lg relative"
    >
      <div
        className="flex flex-col justify-center items-center"
      >
        <h3
          className="py-5 px-4 lg:py-5 lg:px-11 leading-normal text-NeutralBlack lg:text-2xl text-2xl font-regular font-poppins capitalize "
        >
          request submitted
        </h3>
        <div
          className=""
        >
          <h3
            className="py-5 px-4 lg:py-5 lg:px-11 leading-normal text-NeutralBlack lg:text-2xl text-2xl font-regular font-poppins capitalize "
          >
            you will receive a confirmation email soon.
          </h3>
        </div>
      </div>
      <div
        className=" self-center py-10 group "
      >
        <a
          className="w-28 h-10 px-5 py-2 rounded-md text-base font-poppins font-regular bg-Accent text-NeutralBlack group-hover:bg-[#879AB8] group-hover:text-NeutralWhite group-hover:scale-105 duration-500"
          href="/calendar"
          onClick={[Function]}
          onMouseEnter={[Function]}
          onTouchStart={[Function]}
        >
          Back to Calendar
        </a>
      </div>
    </div>
  </div>
</div>
`;
